pkg/ds/daemon_set.go

```diff
@@ -187,7 +187,7 @@ func (ds *daemonSet) WatchDesires(
 					ds.CancelReplication()
 					continue
 				}
-				err := ds.removePods()
+				err = ds.removePods()
```

Comment:
This is a bug, this is in its own namespace and will not be reported in the 
```
			if err != nil {
				select {
				case errCh <- err:
				case <-quitCh:
					return
				}
			}
```
above.